tests/antithesis/test-template/Dockerfile

```diff
@@ -0,0 +1,26 @@
+ARG GO_VERSION=1.24.2
+ARG ARCH=amd64
+
+FROM golang:$GO_VERSION AS build
+
+# Copy replace directives for build
+# TODO - there should be a way to automate this
+ARG BUILD_DIR=/building
```

Comment:
First time see `/building` directory.

Maybe `/build`? Or any more standard name.

Reply:
Done.  I had it as `/build` and then it was conflicting with the build image name.  I have changed the build image name to `builder` instead.